.travis.yml

```diff
@@ -25,9 +16,17 @@ matrix:
       language: generic
       env: TOXENV=py35 BDIST=1
     - python: 3.5
-      env: TOXENV=py35 BDIST=1
+      env: TOXENV=py35 BDIST=1 OPENSSL_OLD
```

Comment:
This should not have BDIST, because then you'll have two different matrix items running pyinstaller, and thus conflicting on the snapshot server, plus one of them won't have functioning ALPN.